### DISCUSSION TITLE
No calendar sync after updating to Thunderbird V91.5.0

### DISCUSSION THREAD

**Discussion by atscomms** [kewisch/gdata-provider#362] [Support] [answered]
Calendars appear empty after updating TB and enabling each calendar
Windows 10 64bit machine
Provider V91.0.2

**Comment by linuxuser63** [comment]
Same issue here. Just got updated to TB91.5.0   
Linux 64 bit, kernel 5.15, glibc 2.3 (Mageia 8 distro)
Keep getting error message when Provider tries to sign into Google, though when I check Google the Provider app has permission to access calendar. However, app continually request that I provide it access and Google repeatedly gives 403 error:rate_limit_exceeded.

**Comment by kewisch** [comment, marked as answer]
rate_limit_exceeded should be fixed by now, this was a temporary issue around an update at the time.
